test-cases/performance/ycsb/perf-base.yaml

```diff
@@ -56,6 +56,8 @@ authenticator_user: cassandra
 authenticator_password: cassandra
 authorizer: 'CassandraAuthorizer'
 
+print_kernel_callstack: true
```

Comment:
isn't it now the default ?

Reply:
it is, but perhaps not in that branch...
we could either backport it to `test_default.yaml` instead of this specific test either way

Reply:
let find that change and backport it to the branch you want

Reply:
#5725